ci/tasks/scripts/pre-build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
```

Comment:
This need to be updated with set -e ... ?

Reply:
forgot to push. Done